docs/src/docs/asciidoc/user/tiering.adoc

```diff
@@ -73,13 +73,17 @@ include::{sourcedir310}/ehcache-impl/src/test/java/org/ehcache/docs/Tiering.java
 <2> A shortcut to specify 10 entries.
 <3> Only 10 MB allowed. Eviction will occur when full.
 
+NOTE: Byte based heap resources are deprecated and will be removed in a future version.
+
 ==== Byte-sized heap
 
 For every tier except the heap tier, calculating the size of the cache is fairly easy.
 You more or less sum the size of all byte buffers containing the serialized entries.
 
 When heap is limited by size instead of entries, it is a bit more complicated.
 
+NOTE: Byte based heap resources are deprecated and will be removed in a future version.
```

Comment:
I would move this note to immediately beneath the section header so that the reader sees it before they read the section. I would also change it to either `WARNING:` or `IMPORTANT:` as they render much more visibly.

Reply:
Agreed, Incorporated